README.md

```diff
@@ -10,11 +10,29 @@ A patch to fix Red Storm Entertainment's mistakes (intentional and otherwise). A
 
 ## To Do
 
-- See GitHub issues list for the complete to-do.  Feel free to request modifications and additions there as well.
+See GitHub issues list for the complete to-do. Feel free to request modifications and additions there as well.
+
+## The OpenRVS Registry
+
+Since v1.5, OpenRVS servers send an extra UDP beacon to a web server running
+[openrvs-registry](https://github.com/ijemafe/openrvs-registry). This app tracks
+all known servers, healthchecks them to hide unhealthy serers, and automatically
+adds new servers to the list when the UDP beacon is received.
+
+The registry listens for beacons on UDP port 8080, and listens for server list
+requests on HTTP port 8080 (on the /servers URL).
+
+The IP and port for the deployment can be configured in `openrvs.ini`:
+
+```ini
+[OpenRVS.OpenBeacon]
+RegistryServerIP=64.225.54.237 # the current deployment
```

Comment:
Does # work as a comment symbol in unreal INI files?  I thought only ; and // would work.  If that's the case, we should change this because it's intended as a quote of the config ini file.

Reply:
Whoops, my text editor highlighted the comment so I thought it was valid, haha. Double checking now.

Reply:
1. Semicolons are the only universal comment indicator
2. Later version of INI software support the # sign
3. No versions support //

https://en.wikipedia.org/wiki/INI_file#Comments

Reply:
I went ahead and removed the comment as it's not clear if mid-line comments are allowed in UE2 INI.

Reply:
That's strange, I've definitely used // as comments in INIs without issue.  Could be just an Unreal thing.